tests/test_distance.py

```diff
@@ -15,12 +15,12 @@
 
 def test_same_graph():
     """The distance between two equal graphs must be zero."""
-    G = nx.karate_club_graph()
+    G = nx.barbell_graph(10, 5)
 
     for label, obj in distance.__dict__.items():
         if isinstance(obj, type) and BaseDistance in obj.__bases__:
             dist = obj().dist(G, G)
-            assert np.isclose(dist, 0.0)
+            assert np.isclose(dist, 0.0), f"{label} not deterministic"
```

Comment:
Nit: I'd rather this just read `label fails same-graph test`. It may be non deterministic, or may be just wrong or buggy or have precision errors...

Reply:
Yeah I didn't like that label anyway